druntime/src/core/internal/newaa.d

```diff
@@ -0,0 +1,144 @@
+/**
+   Turn an Associative Array into a binary compatible struct for static initialization.
+
+   This does not implement all the pieces of
+   the associative array type in druntime, just enough to create an AA from an
+   existing range of key/value pairs.
+
+   Copyright: Copyright Digital Mars 2000 - 2015, Steven Schveighoffer 2022.
```

Comment:
Digital Mars copyright should be dlang foundation I *think*, and the years potentially wrong?

Reply:
This is copied verbatim from here: https://github.com/dlang/dmd/blob/98833b0b82b5f51c769e460c6956d6477480d6c6/druntime/src/rt/aaA.d